config/util/representation-conversion/default.json

```diff
@@ -37,17 +38,28 @@
               "contentType": "text/markdown"
             },
             {
-              "comment": "Renders Markdown snippets into the main page template.",
-              "@type": "MarkdownToHtmlConverter",
+              "comment": "Converts a container into a Markdown listing of its contents.",
+              "@type": "ContainerToTemplateConverter",
               "templateEngine": {
-                "@id": "urn:solid-server:default:MainTemplateEngine",
                 "@type": "HandlebarsTemplateEngine",
-                "template": "$PACKAGE_ROOT/templates/main.html.hbs"
-              }
-            }
+                "template": "$PACKAGE_ROOT/templates/error/container.md.hbs"
+              },
+              "contentType": "text/markdown",
+              "identifierStrategy": { "@id": "urn:solid-server:default:IdentifierStrategy" }
+            },
+            { "@id": "urn:solid-server:default:MarkdownToHtmlConverter" }
           ]
         }
       ]
+    },
+    {
+      "comment": "Renders Markdown snippets into the main page template.",
+      "@id": "urn:solid-server:default:MarkdownToHtmlConverter",
```

Comment:
This can be put in the `config/util/representation-conversion/errors/` folder with the other ones. Might as well put the ContainerToTemplateConverter there as well.

Reply:
Why in errors? This one is not specific to errors.

Reply:
Because I meant `config/util/representation-conversion/converters/` 😅

Reply:
yep!

Reply:
And done.